src/lib/hash/blake2s/blake2s.cpp

```diff
@@ -46,9 +46,8 @@ std::string BLAKE2s::name() const {
 //      (keylen=0: no key)
 
 void BLAKE2s::state_init(size_t outlen, const uint8_t* key, size_t keylen) {
-   for(size_t i = 0; i < 8; i++) {  // state, "param block"
-      m_h[i] = blake2s_iv[i];
-   }
+   std::copy_n(blake2s_iv.data(), 8, m_h.data());  // state, "param block"
+
```

Comment:
For some reason GitHub doesn't let me comment on lines that are unchanged. This is meant to refer to three lines below, the initialization of `m_t`:

Now that `m_t` is a `std::array` we might as well call its constructor:

```C++
m_t = {0};
```